summary.sh

```diff
@@ -30,7 +30,7 @@ echo ""
 echo "${timed_out_list}" | tr ' ' '\n'
 echo ""
 
-unaccounted_for=$(grep -Zvl "no build file found for" "${target}/*.log" \
+unaccounted_for=$(grep -Zvl "no build file found for" "${targetdir}/*.log" \
     | xargs -0 grep -Zvl "dljc could not run the Checker Framework" \
```

Comment:
This code is duplicated in summary.sh and in run-dljc.sh.  Does it need to be in both places?